src/app/shared/helpers/config.ts

```diff
@@ -30,6 +30,8 @@ export interface Environment {
   authType: 'tokens' | 'cookies',
 
   itemPlatformId: string,
+
+  theme: string,
```

Comment:
Since the `theme` here is required to be a string, how is represented the "default" theme ?
In other words: in the PR I only saw the usage with a provided theme, how will it work without a specific theme ?

Reply:
Actually, there is no value for standard theme, because variables for standard theme declared on root lvl. For clear, I can add 'default' value and check it in condition before put to tag. But for now any value which not matched to declared data-theme=[name] would be default theme

Reply:
Ok, then I think the `theme` should be nullable, or required with specific themes like `theme: 'default' | 'client-demo'`